### PR TITLE
Add configuration field for collector's endpoint

### PR DESCRIPTION
**Pull request by shirmoran**
Configuration field for collector's endpoint was added to `tnf_config.yaml`.
In addition, prod collector's DNS was set as the default endpoint.

### CODE REVIEW

File: pkg/certsuite/certsuite.go [hunk 1]
```diff
@@ -48,7 +48,7 @@ func LoadChecksDB(labelsExpr string) {
 
 const (
 	junitXMLOutputFileName = "cnf-certification-tests_junit.xml"
-	collectorAppEndPoint   = "http://44.195.143.94"
+	collectorAppDNS        = "http://claims-collector.cnf-certifications.sysdeseng.com"
```
Comment (rdavid) [inline pkg/certsuite/certsuite.go:51]: DNS stands for Domain Name Service. Did you mean `collectorAppName`?

Comment (shirmoran) [reply pkg/certsuite/certsuite.go:51]: I meant the DNS, as this URL is using the DNS and not the IP

Comment (rdavid) [reply pkg/certsuite/certsuite.go:51]: @shirmoran DNS is a service that is run by a server. Perhaps you meant URL?

Comment (bnshr) [reply pkg/certsuite/certsuite.go:51]: Yeah, `collectorAppUrl` is a better name.

**Review by bnshr — APPROVED** [review]

**Review by sebrandon1 — APPROVED** [review]

**Pull request merged**